src/main/scala/com/databricks/labs/overwatch/pipeline/GoldTransforms.scala

```diff
@@ -950,6 +951,113 @@ trait GoldTransforms extends SparkSessionWrapper {
     df.select(warehouseCols: _*)
   }
 
+  protected def buildWarehouseStateFact(
+                                       instanceDetailsTarget: PipelineTable,
+                                       warehouseSpec: PipelineTable,
+                                       pipelineSnapTime: TimeTypes
+                                     )(warehouseStateDetail: DataFrame): DataFrame = {
+    val data = Seq(
+      ("AWS","2X-Small", "i3.2xlarge", 1, 4),
+      ("AWS","X-Small", "i3.2xlarge", 2, 6),
+      ("AWS","Small", "i3.4xlarge", 4, 12),
+      ("AWS","Medium", "i3.8xlarge", 8, 24),
+      ("AWS","Large", "i3.8xlarge", 16, 40),
+      ("AWS","X-Large", "i3.16xlarge", 32, 80),
+      ("AWS","2X-Large", "i3.16xlarge", 64, 144),
+      ("AWS","3X-Large", "i3.16xlarge", 128, 272),
+      ("AWS","4X-Large", "i3.16xlarge", 256, 528),
+      ("AZURE","2X-Small","E8ds v4",1,4),
+      ("AZURE","X-Small","E8ds v4",2,6),
+      ("AZURE","Small","E16ds v4",4,12),
+      ("AZURE","Medium","E32ds v4",8,24),
+      ("AZURE","Large","E32ds v4",16,40),
+      ("AZURE","X-Large","E64ds v4",32,80),
+      ("AZURE","2X-Large","E64ds v4",64,144),
+      ("AZURE","3X-Large","E64ds v4",128,272),
+      ("AZURE","4X-Large","E64ds v4",256,528)
+    )
```

Comment:
It looks like you have duplicated the data that is also in `src/main/resources/Warehouse_DBU_Details.csv`, correct?  why not load it from the resource file in the JAR (assuming that the resource files are included in the assembly JAR)?  `scala.io.Source.fromResource` from the standard library should be the natural way to do this.

Reply:
@neilbest-db this is done pls check